swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
+from swap_meet import item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory or []
+    
+    def add(self, one_item):
+        if not one_item in self.inventory:
+            self.inventory.append(one_item)
+        return one_item
+    
+    def remove(self, one_item):
+        if not one_item in self.inventory:
+            return False
+        elif one_item in self.inventory:
```

Comment:
Hm, in the `if`, you're checking if `one_item in self.inventory` is False, and in this `elif`, you're checking if it's True. Could we make this `elif` into an `else`? In what situation would we need this to be an `elif` specifically?